### PR TITLE
Fix android problems

### PR DESCRIPTION
**Pull request by alejandro-isaza**
Android is failing to compile, this fixes these problems.

### CODE REVIEW

File: android/app/build.gradle [hunk 1]
```diff
@@ -1,10 +1,9 @@
-apply plugin: 'com.android.application'
+apply plugin: 'com.android.library'
```
Comment (madcake) [inline android/app/build.gradle]: No it's application

File: android/app/build.gradle [hunk 2]
```diff
@@ -1,10 +1,9 @@
...
 
 android {
     compileSdkVersion 28
     buildToolsVersion '28.0.3'
     defaultConfig {
-        applicationId "com.trustwallet.core.app"
```
Comment (madcake) [inline android/app/build.gradle]: It's need

File: android/app/build.gradle (hunk 1 above)
Comment (alejandro-isaza) [reply android/app/build.gradle]: why? it works like this

Comment (madcake) [reply android/app/build.gradle]: App module is application.

Comment (alejandro-isaza) [reply android/app/build.gradle]: ok, will rename to lib

**Pull request merged**